website/src/controllers/api-v3/content.js

```diff
@@ -24,12 +73,33 @@ api.getContent = {
   method: 'GET',
   url: '/content',
   async handler (req, res) {
-    let language = req.query.language ? req.query.language.toString() : 'en';
-    let content = _.cloneDeep(common.content);
+    let language = 'en';
+    let proposedLang = req.query.language && req.query.language.toString();
+
+    if (proposedLang && _.isBoolean(cachedContentResponses[proposedLang])) {
```

Comment:
You can also do:

```js
if (proposedLang && proposedLang in cachedContentResponses) {
```

Which I think is more readable.

In fact, I think you _could_ just do:

```js
if (proposedLang in cachedContentResponses) {
```

But I'd want to double check that.